src/cli.ts

```diff
@@ -147,14 +148,20 @@ program
 // Push command
 program
   .command('push')
-  .addArgument(
-    new Argument(
-      '[journeys...]',
-      'file path to journey directory and individual files'
-    ).argRequired()
-  )
   .description(
-    'Push journeys to create montors within Kibana monitor management UI'
+    'Push journeys in the current directory to create montors within the Kibana monitor management UI'
+  )
+  .option(
```

Comment:
Not sure why we need these options here? Subcommands should be able to access these `pattern, tags, match` without any problem IIRC.